PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+## Description
+
+What was changed in this pull request?
+
+Why is it necessary?
+
```

Comment:
Perhaps ask for a link to an issue or Slack discussion related to the PR and what it addresses?